lib_eio_linux/eio_linux.ml

```diff
@@ -537,8 +537,8 @@ let run_event_loop (type a) ?fallback config (main : _ -> a) arg : a =
       | Eio_unix.Private.Pipe sw -> Some (fun k ->
           match
             let r, w = Low_level.pipe ~sw in
-            let r = (Flow.of_fd r :> _ Eio_unix.source) in
-            let w = (Flow.of_fd w :> _ Eio_unix.sink) in
+            let r = (Flow.of_fd r :> [< Eio_unix.source_ty] Eio.Std.r) in
```

Comment:
I actually think we can remove the explicit conversions here no?

```suggestion
            let r = Flow.of_fd r in
```

Reply:
Thank you for this review!
For the explicit conversions, I'd suggest we use it since we're dealing with cases where polymorphism is needed. 
And also, I just generalized the values of `r `and `w`.

I think the base benefit of removing it is just that it makes the code cleaner.

Reply:
Though I could replace it with these

```
let r = Flow.of_fd r in
let w = Flow.of_fd w in
```